mne/preprocessing/ica.py

```diff
@@ -181,8 +183,8 @@ class ICA(ContainsMixin):
         np.random.RandomState to initialize the FastICA estimation.
         As the estimation is non-deterministic it can be useful to
         fix the seed to have reproducible results. Defaults to None.
-    method : {'fastica', 'infomax', 'extended-infomax'}
-        The ICA method to use. Defaults to 'fastica'.
+    method : {'fastica', 'infomax', 'extended-infomax', 'picard'}
+        The ICA method to use. Defaults to 'fastica'. For reference, see [1-4].
```

Comment:
For proper sphinx/HTML links use
```
see [1]_, [2]_, [3]_, [4]_.
```
maybe there is some syntax to combine them (feel free to check) but what I wrote should work